Source/Core/Common/WindowSystemInfo.h

```diff
@@ -13,6 +13,7 @@ enum class WindowSystemType
   Wayland,
   FBDev,
   Haiku,
+  DRM,
```

Comment:
I'm not sure why this is highlighted in blue, its not a C++ keyword. Anyone know?